pkg/fixtures/deployments/helm/charts/values.yaml

```diff
@@ -54,10 +44,62 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
+livenessProbe:
+  tcpSocket:
+    port: 80
+readinessProbe:
+  tcpSocket:
+    port: 80
+  periodSeconds: 5
+  timeoutSeconds: 5
+  failureThreshold: 1
+  successThreshold: 1
+  initialDelaySeconds: 3
+
 nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
+topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: ScheduleAnyway
+    labelSelector:
+      matchLabels:
+        app.kubernetes.io/name: testapp
+
+affinity:
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - weight: 100
+      podAffinityTerm:
+        topologyKey: kubernetes.io/hostname
+        labelSelector:
+          matchLabels:
+            app.kubernetes.io/name: testapp
+
+securityContext:
+  seccompProfile:
+    type: RuntimeDefault
+  capabilities:
+    drop:
+      - ALL
+    add:
```

Comment:
why are we adding all these?

Reply:
From the PRD! There were also some runAsNonRoot fields as well but they blocked a few of our int tests(rust/erlaang/swift) so ill investigate those at a later date